app/components/Metric.jsx

```diff
@@ -1,10 +1,10 @@
 import React, { Component } from 'react';
 import MetricTemplate from './MetricTemplate.jsx';
 
+// Renders Metric component
```

Comment:
This comment isn't too descriptive, and it's also not really the Metric class that is rendering stuff, but rather a method within the Metric class. Instead, maybe it would be more helpful to describe what the Metric class actually represents?